src/inmanta/data/model.py

```diff
@@ -316,6 +319,31 @@ def level_from_enum_attribute(cls, v: object) -> object:
     kwargs: Dict[str, Optional[ArgumentTypes]] = {}
     timestamp: datetime.datetime
 
+    def dict(
+        self,
+        *,
+        include: Optional[Union["AbstractSetIntStr", "MappingIntStrAny"]] = None,
+        exclude: Optional[Union["AbstractSetIntStr", "MappingIntStrAny"]] = None,
+        by_alias: bool = False,
+        skip_defaults: Optional[bool] = None,
+        exclude_unset: bool = False,
+        exclude_defaults: bool = False,
+        exclude_none: bool = False,
+    ) -> "DictStrAny":
+        generated_dict = super().dict(
+            include=include,
+            exclude=exclude,
+            by_alias=by_alias,
+            skip_defaults=skip_defaults,
+            exclude_unset=exclude_unset,
+            exclude_defaults=exclude_defaults,
+            exclude_none=exclude_none,
+        )
+
+        # Overwrite the value of the level with its name as the db expects this
```

Comment:
how is this related to the DB? This is the data transfer object?

Reply:
Here: https://github.com/inmanta/inmanta-core/blob/master/src/inmanta/server/services/resourceservice.py#L460
And I wasn't sure if is wasn't being used the same way in other places so I figure I would change it at the root.  I can try changing it only there and see if it works.

Reply:
This change made this test fail, it really want to get the level name when serializing the logline.  Is there a reason for that? https://github.com/inmanta/inmanta-core/blob/master/tests/test_data_model.py#L74
Or can I change the test?